apps/mymedicare_cb/fixtures.py

```diff
@@ -0,0 +1,50 @@
+patient_response = {
```

Comment:
This is test only, right? Would it make sense to make a test folder and move this there?